### PR TITLE
Component refactoring

### PR DESCRIPTION
**Pull request by eberger727**
- Icons are white
- Formatted Contract Deploy Stepper
- Remove upper case text in button and tabs
- Consistent spacing
- Renaming headers
- Instruction modal
- Localization for text
- Current state refresh bug

Screen Shot 2022-04-25 at 2 37 48 PM

https://user-images.githubusercontent.com/17093033/165152829-47af55aa-2b7d-4a65-98a6-c7cb488f56b7.mov

### CODE REVIEW

**Comment by lanasta** [comment]
🔥

**Comment by awrichar** [comment]
Does this resolve #66?

File: ui/src/constants/TutorialSections.tsx [hunk 1]
```diff
@@ -116,6 +137,7 @@ export const TutorialSections: ITutorialSection[] = [
         runnable: true,
         shortInfo: 'burnShortInfo',
         title: 'burnTitle',
+        icon: <LocalFireDepartmentOutlined />,
```
Comment (awrichar) [inline ui/src/constants/TutorialSections.tsx:140]: 👨‍🚒

**Comment by eberger727** [comment]
> Does this resolve #66?
Screen Shot 2022-04-25 at 3 53 51 PM
now it does

**Review by awrichar — APPROVED** [review]

**Pull request merged**